components/basecomponents/KidsTranslationContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useTranslation } from 'next-i18next';
+import Image from 'next/image';
+import { KidsTranslation } from './KidsTranslation';
+import { Language } from '../../data/locales';
+
+export interface Translation {
+  cz_translation: string;
+  ua_translation: string;
+  ua_transcription: string;
+  cz_transcription: string;
+}
+
+interface TranslationContainerProps extends Translation {
+  searchText?: string;
+  image: string;
+  setPlayer: React.Dispatch<React.SetStateAction<HTMLAudioElement | null>>;
+  player: HTMLAudioElement | null;
+}
+
+/**
+ *  Displays list of translations in opened collapse component
+ *
+ * @returns
+ */
+export const KidsTranslationsContainer = ({
+  cz_translation,
+  ua_translation,
+  ua_transcription,
+  cz_transcription,
+  image,
+  setPlayer,
+  player,
+}: TranslationContainerProps): JSX.Element => {
+  const { i18n } = useTranslation();
+
+  const currentLanguage = i18n.language as Language;
+  const secondaryLanguage: Language = currentLanguage === 'uk' ? 'cs' : 'uk';
+
+  const languageTranslation = {
```

Comment:
This is a copy of the same const in Translation container. Again, you can extract it into a separate file and use the same one in both components. Later, in a separate pull request, we should refactor this functionality into its own class instead of relying on this object.